File: CursoPOOUber/Java/UberPool.java
package CursoPOOUber.Java;
class UberPool extends Car {
    String brand;
    String model;

    public UberPool(String license, Account driver, String brand, String model){
        super(license,driver); // Atributos de la super clase en este caso seria car seria 
        this.brand = brand; // Atributos de la clase uberx
        this.model = model;

    }
}

File: CursoPOOUber/Pyhton/uberPool.py
from car import Car

class UberPool(Car):
    brand = str
    model = str

    def __init__(self, license, driver, brand, model):
        super.__init__(license,driver) #Linea de la herenciade car.py
        self.brand= brand #Atributos de la clase uberpool
        self.model= model
